Assets/GothicVR/Scripts/UI/TextureManager.cs

```diff
@@ -0,0 +1,59 @@
+using System;
+using UnityEngine;
+using GVR.Caches;
+using GVR.Util;
+using GVR.Phoenix.Interface;
+using TMPro;
+public class TextureManager : MonoBehaviour
+{
+    private Texture2D backgroundtexture;
+    public Material backgroundmaterial;
+    private Texture2D buttontexture;
+    public Material buttonmaterial;
+    private Texture2D slidertexture;
+    public Material slidermaterial;
+    private Texture2D sliderpositiontexture;
+    public Material sliderpositionmaterial;
+    private Texture2D arrowtexture;
+    public Material arrowmaterial;
+    private Texture2D fillertexture;
+    public Material fillermaterial;
+    private Texture2D skytexture;
+    public Material skymaterial;
+    bool textureloaded = false;
+
+    // Update is called once per frame
+    void Update()
+    {
+        if (!textureloaded && GameData.I.VdfsPtr != IntPtr.Zero)
```

Comment:
Fine for now. I referenced it inside this ticket to be improved: https://github.com/GothicVRProject/GothicVR/issues/80

Rule of thumb: We should see if we never use Update() if not really neccessary. ;-)

Reply:
True. Thanks.